CMSgov/mint-app
# [EASI-2700] Upgrade to storybook 7

# EASI-2700

## Changes and Description

- Upgrades Storybook and modifies necessary configuration so that it can run locally
- Modifies Tabs SASS files so that they properly compile for Storybook (these components aren't used in the app, so their incorrect SASS wasn't caught until we tried to compile them in Storybook)

## How to test this change

- Run `yarn run storybook` and ensure the storybook dev server works as expected

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
